boot/bootutil/include/bootutil/enc_key.h

```diff
@@ -45,7 +46,17 @@ struct enc_key_data {
     bootutil_aes_ctr_context aes_ctr;
 };
 
-extern const struct bootutil_key bootutil_enc_key;
+/**
+ * Retrieve the private key for image encryption.
+ *
+ * @param[out]  private_key  structure to store the private key and
+ *                           its length.
+ *
+ * @return                   0 on success; nonzero on failure.
+ *
+ */
+int boot_retrieve_private_key(struct bootutil_key **private_key);
```

Comment:
Would you mind renaming this function to `boot_enc_retrieve_priv_key` or similar?
It's for avoiding ambiguity - one might think that it's retrieving the private key used for image signing (there's already a function called `boot_retrieve_public_key_hash`).

Reply:
The renaming is still missing in one file. I've added a comment to it.